middleware/http/sentinel/middleware_test.go

```diff
@@ -0,0 +1,49 @@
+package sentinel
+
+import (
+	"testing"
+
+	"github.com/dapr/components-contrib/middleware"
+	"github.com/dapr/dapr/pkg/logger"
```

Comment:
This has moved to `"github.com/dapr/kit/logger"`

Reply:
fixed.